### DISCUSSION TITLE
'as' prop in <Link> missing in Next13?

### DISCUSSION THREAD

**Discussion by williamlmao** [vercel/next.js#42060] [Help] [answered]
I combed through the documentation for Next 13 and didn't find an equivalent to the 'as' property. I think this is an important feature, because it allows for [contextual routing](https://dev.to/toomuchdesign/contextual-routing-and-modal-routes-in-next-js-18fn). Does anyone know of equivalent functionality in next13? 

[Next 12 docs](https://nextjs.org/docs/api-reference/next/link)
> as - Optional decorator for the path that will be shown in the browser URL bar. Before Next.js 9.5.3 this was used for dynamic routes, check our [previous docs](https://nextjs.org/docs/tag/v9.5.2/api-reference/next/link#dynamic-routes) to see how it worked. Note: when this path differs from the one provided in href the previous href/as behavior is used as shown in the [previous docs](https://nextjs.org/docs/tag/v9.5.2/api-reference/next/link#dynamic-routes).

**Comment by delbaoliveira** [comment, marked as answer]
Hey @williamlmao, we're planning to add routing patterns which will replace the behaviour of `as`. 

I've added a note to the docs here:
https://beta.nextjs.org/docs/api-reference/components/link#legacy-props

**Reply by ciruz** [reply]
Is there any workaround for now for this, when using the ``/app`` folder? Thanks
